orm/model_bucket.go

```diff
@@ -34,7 +34,16 @@ type ModelBucket interface {
 	// is returned.
 	One(db weave.ReadOnlyKVStore, key []byte, dest Model) error
 
-	// Put saves given model in the database.
+	// Many returns all objects that secondary index with given name is
+	// given key. Main index is always unique but secondary indexes can
+	// return more than one value for the same key.
+	// All matching entities are appended to given destination slice. If no
+	// result was found, no error is retured and destination slice is not
+	// modified.
+	Many(db weave.ReadOnlyKVStore, indexName string, key []byte, dest *[]Model) error
```

Comment:
`Many` seems a bit odd here, I would use `ByIndex` or such.

`Many` for me would be more like the return value of a Range (on the primary key)

Reply:
Function definition (API) looks good though, besides the naming comment